sdk/eventhub/event-hubs/README.md

```diff
@@ -1,259 +1,198 @@
-or any other IDE that provides better intellisense and exposes the full power of source code documentation.
+**Prerequisites**: You must have an [Azure subscription](https://azure.microsoft.com/free/) and a
+[Event Hubs Namespace](https://docs.microsoft.com/en-us/azure/event-hubs/) to use this package.
+If you are using this package in a Node.js application, then use Node.js 6.x or higher.
 
-## Debug logs ##
+### Configure Typescript
 
-You can set the following environment variable to get the debug logs.
+TypeScript users need to have Node type definitions installed:
 
-- Getting debug logs from the Event Hub SDK
 ```bash
-export DEBUG=azure*
-```
-- Getting debug logs from the Event Hub SDK and the protocol level library.
-```bash
-export DEBUG=azure*,rhea*
-```
-- If you are **not interested in viewing the message transformation** (which consumes lot of console/disk space) then you can set the `DEBUG` environment variable as follows:
-```bash
-export DEBUG=azure*,rhea*,-rhea:raw,-rhea:message,-azure:amqp-common:datatransformer
-```
-- If you are interested only in **errors**, then you can set the `DEBUG` environment variable as follows:
-```bash
-export DEBUG=azure:event-hubs:error,azure-amqp-common:error,rhea-promise:error,rhea:events,rhea:frames,rhea:io,rhea:flow
+npm install @types/node
 ```
 
-#### Logging to a file
-- Set the `DEBUG` environment variable as shown above and then run your test script as follows:
-  - Logging statements from your test script go to `out.log` and logging statements from the sdk go to `debug.log`.
-    ```bash
-    node your-test-script.js > out.log 2>debug.log
-    ```
-  - Logging statements from your test script and the sdk go to the same file `out.log` by redirecting stderr to stdout (&1), and then redirect stdout to a file:
-    ```bash
-    node your-test-script.js >out.log 2>&1
-    ```
-  - Logging statements from your test script and the sdk go to the same file `out.log`.
-    ```bash
-      node your-test-script.js &> out.log
-    ```
+You also need to enable `compilerOptions.allowSyntheticDefaultImports` in your tsconfig.json. Note that if you have enabled `compilerOptions.esModuleInterop`, `allowSyntheticDefaultImports` is enabled by default. See [TypeScript's compiler options handbook](https://www.typescriptlang.org/docs/handbook/compiler-options.html) for more information.
+
+### Authenticate the client
+
+Interaction with Event Hubs start with an instance of the [EventHubClient](https://docs.microsoft.com/en-us/javascript/api/%40azure/event-hubs/eventhubclient?view=azure-node-latest) class. You can instantiate
+this class using one of the 4 static methods on it
 
-## Examples ##
+- [createFromConnectionString](https://docs.microsoft.com/en-us/javascript/api/%40azure/event-hubs/eventhubclient?view=azure-node-latest#createfromconnectionstring-string--ClientOptions-)
```

Comment:
The link here for some reason is not landing on the right place. Can you take a look?